scripts/drupal-vm/Vagrantfile

```diff
@@ -4,7 +4,7 @@ ENV['DRUPALVM_PROJECT_ROOT'] = "#{__dir__}"
 
 # The relative path from the project root to the config directory where you
 # placed your config.yml file.
-ENV['DRUPALVM_CONFIG_DIR'] = "box"
+ENV['DRUPALVM_CONFIG_DIR'] = "${drupalvm.config.dir}"
```

Comment:
So this needs to be a relative path, and that config value (`blt.config-files.dir`) is the absolute path when calculated, I believe. I create the relative path in the `Vagrantfile` copy command, and set this variable just for the purposes of this command.